content/news/2022.pml

```diff
@@ -55,6 +55,34 @@ Older News: <%=reflist%>
 -->
 
 
+<div class="news-item" id="member-ncsc">
+  15 June 2022: LatticeX joins the seL4 Foundation
+</div>
+<div>
+  <p>
+    <a href="https://latticex.foundation/home">
+      <img src="/Foundation/Membership/LOGOS/latticex.png"
+	   style="width: 20%;  padding-left:10px;  float:right"
+	   alt="LatticeX" />
+    </a>
+    The seL4 Foundation is pleased to welcome <a href="https://latticex.foundation/home">Lattice X</a>.
+    <br>
+    <br>
+    LatticeX Foundation cultivates academic research, technology advancements and industrial applications in the fields that foster the development of LatticeX ecosystem.
+    <br>
+    <br>
+    LatticeX is the vision of large-scale distributed network supporting the expansive economy activities and broader business applications where users can recognize the sovereignty and privacy of their own data, and leverage the ecosystem of sophisticated computing technologies to collaboratively unearth the value of data and benefit from data-value transactions throughout.
+    <br>
+    <br>
+    "We are so glad to join seL4 foundation. Hopefully, we can specialise seL4 technology into the web 3.0 domain from cross chain bridge node OS to hardware wallet OS for improving their security and stability."
```

Comment:
shouldn't this quote mention a person from Lattice X who made it?

Reply:
I've added Yuning's name and role.